SetupDataPkg/Library/ConfigKnobShimLib/ConfigKnobShimDxeLib/ConfigKnobShimDxeLib.inf

```diff
@@ -38,4 +38,4 @@
 
 [Depex]
   # Platforms can decide whether variable services are a requirement for config or not
-  gEfiVariableArchProtocolGuid
+  gEfiVariableArchProtocolGuid
```

Comment:
not sure why the diff looks like this. There was a newline and now there's not

Reply:
Should be a small fix to add it back

Reply:
> Should be a small fix to add it back

Add it back in the .inf? This was addressing a comment on the old pr you had that I thought was suggesting to remove the newline (https://github.com/microsoft/mu_feature_config/pull/200#discussion_r1228712196)

So all files should have a newline?

Reply:
Yeah, all the files should have a newline at EOF